psyneulink/library/compositions/autodiffcomposition.py

```diff
@@ -281,16 +274,10 @@ def __init__(self,
     def _build_pytorch_representation(self, context=None):
         if self.scheduler is None:
             self.scheduler = Scheduler(graph=self.graph_processing)
-        if self.execution_sets is None:
-            self.execution_sets = [ x - set(self.get_nodes_by_role(NodeRole.LEARNING)) for x in self.scheduler.run(context=context)]
-            self.execution_sets = [x for x in self.execution_sets if len(x) > 0]
-        if self.parameters.pytorch_representation._get(context) is None:
-            model = PytorchModelCreator(self.graph_processing,
-                                        self.param_init_from_pnl,
-                                        self.execution_sets,
-                                        self.device,
+        if self.parameters.pytorch_representation._get(context=context) is None:
+            model = PytorchModelCreator(composition = self,
+                                        device = self.device,
```

Comment:
arg assignment should be surrounded by whitespace

Reply:
sorry that should should ahve been "should not be surrounded by whitespace". sorry for the confusion.
basically following https://www.flake8rules.com/rules/E251.html, in before it can be enforced globally